CHANGELOG.md

```diff
@@ -8,6 +8,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Added
 - Added support for row and column reordering. (#343)
+- Added type inferrence for subtypes for number. (#313)
+- Added parsing of number literals containing '%' or currency symbol (default '$').
```

Comment:
Please add the issue/PR number

Reply:
done

Reply:
we're not really explaining many aspects of metadata, e.g. automatic coercions of arguments. The missing part is part of `runFunction` wrapper, which is not mandatory for usage. The same for `returnNumberType` and `passSubtype` -- those are from `runFunction` wrapper.

Reply:
Yup. We have an issue #557 to update the docs after the `runFunction` was introduced